homeassistant/components/device_tracker/__init__.py

```diff
@@ -96,9 +91,6 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     "SourceType",
     "TrackerEntity",
     "TrackerEntityDescription",
-    "async_setup",
     "async_setup_entry",
     "async_unload_entry",
-    "is_on",
-    "see",
```

Comment:
Don't remove things that were imported here before the other PR.

I agree it's better to revert the other PR and start over.


Many of these names are used by legacy device tracker platforms, and we still need to support them.